### PR TITLE
added docker

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+.git
+.vscode
+
+*Dockerfile*
+*docker-compose*
+
+node_modules
+
+*.md
+*.log
```
Comment (asafMasa) [inline .dockerignore:11]: enter

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM node:12.18.1-alpine3.11 as production
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
+COPY . .
+EXPOSE 3009
+CMD ["node", "index.js"]
```
Comment (asafMasa) [inline Dockerfile:7]: enter

Comment (CL-SHLOMIKONCHA) [reply Dockerfile]: fixed in 9d7e1a2

File: .dockerignore (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply .dockerignore]: fixed in 9d7e1a2

**Pull request merged**